### DISCUSSION TITLE
Estimated camera parameters

### DISCUSSION THREAD

**Discussion by jamesyoung0623** [Arthur151/ROMP#447] [General]
Hello,
Thank you for the great work!
I want to ask how to construct the intrinsic and extrinsic parameters from the data in saved npz files.
There are items like cam, global_orient, center_preds, center_confs, and cam_trans but I don't know how to construct the intrinsic and extrinsic parameters with these data.
Can someone help?
Thanks in advance.

**Comment by KaneLindsay** [comment]
I would also like to know this. Did you figure it out @jamesyoung0623? :octocat:
